bin/plugin/config.js

```diff
@@ -11,6 +11,7 @@
 const config = {
 	githubRepositoryOwner: 'WordPress',
 	githubRepositoryName: 'performance',
+	textdomain: 'performance-lab',
```

Comment:
To be consistent with the rest of the config this one should be:

```suggestion
	textDomain: 'performance-lab',
```